templates/ilvo/ILVO libprep.json

```diff
@@ -0,0 +1,318 @@
+            "FAIRE-seq",
+            "SELEX",
+            "RIP-Seq",
+            "ChIA-PET",
+            "Synthetic-Long-Read",
+            "Targeted-Capture",
+            "OTHER"
+          ]
+        },
+        {
+          "iri": null,
+          "name": "library selection",
+          "description": "Whether any method was used to select for or against, enrich, or screen the material being sequenced. (SRA 1.2 documentation)",
+          "dataType": "Controlled Vocabulary",
+          "required": true,
+          "isaTag": "parameter_value",
+          "ontology": null,
+          "CVList": [
+            "RANDOM",
+            "PCR",
+            "RANDOM PCR",
+            "RT-PCR",
+            "HMPR",
+            "MF",
+            "repeat fractionation",
+            "size fractionation",
+            "MSLL",
+            "cDNA",
+            "cDNA_randomPriming",
+            "cDNA_oligo_dT",
+            "PolyA",
+            "Oligo-dT",
+            "Inverse rRNA",
+            "Inverse rRNA selection",
+            "ChIP",
+            "ChIP-Seq",
+            "MNase",
+            "DNase",
+            "Hybrid Selection",
+            "Reduced Representation",
+            "Restriction Digest",
+            "5-methylcytidine antibody",
+            "MBD2 protein methyl-CpG binding domain",
+            "CAGE",
+            "RACE",
+            "MDA",
+            "padlock probes capture method",
+            "other",
+            "unspecified"
+          ]
+        },
+        {
+          "iri": null,
+          "name": "library layout",
+          "description": "specifies whether to expect single, paired, or other configuration of reads.",
+          "dataType": "Controlled Vocabulary",
+          "required": true,
+          "isaTag": "parameter_value",
+          "ontology": null,
+          "CVList": ["SINGLE", "PAIRED"]
+        },
+        {
+          "iri": null,
+          "name": "library strand",
+          "description": "",
+          "dataType": "Controlled Vocabulary",
+          "required": false,
+          "isaTag": "parameter_value",
+          "ontology": null,
+          "CVList": ["first strand", "second strand", "not applicable"]
+        },
+        {
+          "name": "orientation",
+          "description": "",
+          "dataType": "Controlled Vocabulary",
+          "required": false,
+          "isaTag": "parameter_value",
+          "ontology": null,
+          "CVList": ["5'-3'-3'-5'", "5'-3'-5'-3'"]
+        },
+        {
+            "name": "insert size",
+            "description": "",
+            "dataType": "String",
+            "required": false,
+            "isaTag": "parameter_value",
+            "ontology": null,
+            "CVList": null
+        },
+        {
+            "name": "primers remarks",
+            "description": "",
+            "dataType": "String",
+            "required": false,
+            "isaTag": "otherMaterial_characteristic",
```

Comment:
The difference between parameter value and characteristic is always a bit open to interpretation, it depends on the specific protocol, I think.

Reply:
Agreed.
Just trying to match what was originally defined in the other templates and in the ISA configuration file.
This will, however, require a future dedicated discussion